bin/main.rb

```diff
@@ -1,67 +1,86 @@
 #!/usr/bin/env ruby
 
+require_relative '../lib/players'
+require_relative '../lib/game_status'
+
 class Board
   attr_accessor :board
 
   def initialize
     @board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+    @count = 1
+    @game_status = GameStatus.new
   end
 
   def display_board
-    puts "#{board[0]} | #{board[1]} | #{board[2]}\n---------\n#{board[3]} | #{board[4]} | #{board[5]}\n---------
-    \n#{board[6]} | #{board[7]} | #{board[8]}"
+    puts "#{@board[0]} | #{@board[1]} | #{@board[2]}"
+    puts '---------'
+    puts "#{@board[3]} | #{@board[4]} | #{@board[5]}"
+    puts '---------'
+    puts "#{@board[6]} | #{@board[7]} | #{@board[8]}"
   end
-end
 
-class Player
-  attr_reader :name, :sign
-  attr_accessor :record
+  def welcome
+    puts 'Welcome to Tic Tac Toe'
+    puts ''
+  end
```

Comment:
- [ ] No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

- After you put the Board class in the lib file you won't be able to use puts so instead have a String variable where you concatenate each puts and retrieve it at the end, then you can call it in the `main.rb` file like this:
```ruby
.
.
.
game_board = Board.new
puts game_board.welcome
.
.
.
```